tests/qchem/test_tapering.py

```diff
@@ -570,6 +570,7 @@ def test_taper_obs(symbols, geometry, charge, op_arithmetic):
     tapered Hartree-Fock state (:math:`\langle HF|obs|HF \rangle`) are consistent."""
     if op_arithmetic:
         enable_new_opmath()
```

Comment:
Since the default will switch, I think we should check if opmath was enabled or disabled and restore whatever that was after the test is run, else it will affect other tests. E.g.

```
was_active = qml.operation.active_new_opmath()

if op_arithmetic:
    enable_new_opmath()
.. test..
if not was_active:
    disable_new_opmath()
else:
    enable_new_opmath()